tests/test_help_messages.py

```diff
@@ -0,0 +1,28 @@
+from snowflake.cli.app.commands_registration.command_plugins_loader import (
+    load_only_builtin_command_plugins,
+)
+
+
+def test_help_messages(runner, snapshot):
+    """
+    Iterate through all commands and check all their help messages against the snapshot
+    """
+    IGNORE_PLUGINS = ["render"]
+
+    def _invoke_command(command):
```

Comment:
```suggestion
    def _invoke_command_and_check_help_message(command):
```
Or something to be more appropriate?

Reply:
Is "check" got enough? I renamed dfs as well